AdobeIms/view/adminhtml/web/js/action/authorization.js

```diff
@@ -113,11 +113,6 @@ define([
                 if (authWindow.closed) {
                     clearTimeout(stopWatcherId);
                     clearInterval(watcherId);
-
-                    // eslint-disable-next-line max-depth
-                    if (window.adobeIMSAuthWindow && window.adobeIMSAuthWindow.closed) {
-                        deferred.reject(new Error('Authentication window was closed.'));
```

Comment:
@coderimus we have to reject the promise when window is closed even if we wouldn't like to display the error